templates/lot_of_comments.html

```diff
@@ -0,0 +1,27 @@
+<p>The following {{ plural('bug has', data, pword='bugs have') }} many comments:</p>
+<table {{ table_attrs }}>
+    <thead>
+        <tr>
+            <th>Bug</th>
+            <th>Summary</th>
+            <th>Creation</th>
+            <th># of comments</th>
+            <th>Last comment</th>
```

Comment:
The header order does not match the data in the rows: 
```suggestion
            <th>Last comment</th>
            <th># of comments</th>
```

Reply:
oups, thanks